src/parser/jobs/sam/modules/Kenki/Kenki.tsx

```diff
@@ -60,7 +60,7 @@ export class Kenki extends CoreGauge {
 		[this.data.actions.HISSATSU_KYUTEN.id, {action: -25}],
 		[this.data.actions.HISSATSU_SHINTEN.id, {action: -25}],
 		[this.data.actions.HISSATSU_SENEI.id, {action: -25}],
-		[this.data.actions.HISSATSU_KAITEN.id, {action: -20}],
+		[this.data.actions.HISSATSU_KAITEN.id, {action: -20}], //Removed. In theory an unknown action will break this if cast by player.
```

Comment:
Remove it from the list entirely, then in the initialiser, if it's pre-6.1 add it selectively. Maps are nice like that.

Reply:
Change made.